matic/contracts/WildcardSteward_v3_matic.sol

```diff
@@ -1,1226 +0,0 @@
-// NOTE : this is actually the v0 matic contract.
```

Comment:
Deleting this, the file is now called `_matic_v0` (which is our v3 contracts from mainnet deployed to matic basically)